commands/issue.go

```diff
@@ -39,7 +39,15 @@ var (
 `,
 	}
 
+	cmdCreateIssue = &Command{
+		Key:   "create",
+		Run:   createIssue,
+		Usage: "issue create [-m <MESSAGE>|-f <FILE>] [-l <LABELS>]",
```

Comment:
>Usage and Long are both used for generating documentation for a command.

That is not true today for subcommands: either open an issue to make `hub help issue create` display the help for the subcommand, or just remove the doc here.